### PR TITLE
Create problem24.py

### PR DESCRIPTION
**Pull request by Anubhav1107**

### CODE REVIEW

File: python/problem24.py [hunk 1]
```diff
@@ -0,0 +1,7 @@
+#To get the millionth permutation of 0,1,2,3,4,5,6,7,8,9
+
+import itertools
+from itertools import isslice
```
Comment (TheSpeedX) [inline python/problem24.py:4]: You might consider removing this line

Comment (TheSpeedX) [inline python/problem24.py:4]: You might consider removing this line

**Review by Anubhav1107 — COMMENTED** [review]
Removed Line 4, thanks for telling

**Comment by TheSpeedX** [comment]
Thanks

**Pull request merged**